src/publisher/pubsub-message.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {google} from '../../protos/protos';
+
+/**
+ * Strings are the only allowed values for keys and values in message attributes.
+ */
+export interface Attributes {
```

Comment:
This type kinda exists as `Record<string, string>` https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype

Reply:
I'm sliiiightly nervous about changing this at this point simply because it was moved from another file, and is already exposed to users indirectly. But it sounds like it should be pretty equivalent.